schema/src/main/scala/CpgExtSchema.scala

```diff
@@ -82,6 +82,7 @@ class CpgExtSchema(builder: SchemaBuilder, cpgSchema: CpgSchema) {
   property.addOutEdge(edge = isUsedAt, inNode = literal)
   property.addOutEdge(edge = isUsedAt, inNode = methodParameterIn)
   property.addOutEdge(edge = isUsedAt, inNode = member)
+  property.addOutEdge(edge = isUsedAt, inNode = callNode)
```

Comment:
You can remove line number 84 and 85 and replace with 
`property.addOutEdge(edge = isUsedAt, inNode = astNode)`

Reply:
refer how we do for SQLNodes